.travis.yml

```diff
@@ -19,6 +19,9 @@ install:
   - pip install testfixtures
   - pip install unittest2
   - pip install scikit-learn
+  - pip install theano
+  - pip install tensorflow
+  - pip install keras
```

Comment:
The discussion [here](https://github.com/RaRe-Technologies/gensim/pull/1336/files/f0ba459269460fe24d527550f223a90924688faa#r117319253) seems to suggest that we actually use the same list for `test_env` and `tests_require` in `setup.py`. So, I have added these 3 requirements to `test_env`. 
Please let me know if I have misunderstood this.